proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -85,6 +85,9 @@ message GetLogRequest {
   // false: all values returned are properly boxed.
   bool json_values = 2;
 
+  // If set, prints will not be returned.
```

Comment:
Add in the comment an explanation of what this is useful for, maybe even an example.